bndtools.core/src/bndtools/wizards/project/ImportBndWorkspaceWizardPageOne.java

```diff
@@ -0,0 +1,249 @@
+
+        FormData fd_txtFolder = new FormData();
+        fd_txtFolder.top = new FormAttachment(lblFolder, 0, SWT.CENTER);
+        fd_txtFolder.left = new FormAttachment(lblFolder, 10);
+        fd_txtFolder.right = new FormAttachment(100, -100);
+        txtFolder.setLayoutData(fd_txtFolder);
+
+        FormData fd_btnDialog = new FormData();
+        fd_btnDialog.top = new FormAttachment(lblFolder, 0, SWT.CENTER);
+        fd_btnDialog.left = new FormAttachment(txtFolder, 10);
+        fd_btnDialog.right = new FormAttachment(100, -10);
+        btnOpenDialog.setLayoutData(fd_btnDialog);
+
+        FormData fd_lblProjects = new FormData();
+        fd_lblProjects.top = new FormAttachment(lblFolder, 20);
+        fd_lblProjects.left = new FormAttachment(lblFolder, 0, SWT.LEFT);
+        lblProjects.setLayoutData(fd_lblProjects);
+
+        FormData fd_table = new FormData();
+        fd_table.top = new FormAttachment(lblProjects, 5);
+        fd_table.left = new FormAttachment(lblFolder, 0, SWT.LEFT);
+        fd_table.right = new FormAttachment(100, -100);
+        fd_table.bottom = new FormAttachment(100, -10);
+        tableViewer.getTable().setLayoutData(fd_table);
+
+        FormData fd_btnRefresh = new FormData();
+        fd_btnRefresh.top = new FormAttachment(tableViewer.getTable(), 0, SWT.TOP);
+        fd_btnRefresh.left = new FormAttachment(btnOpenDialog, 0, SWT.LEFT);
+        fd_btnRefresh.right = new FormAttachment(100, -10);
+        refreshButton.setLayoutData(fd_btnRefresh);
+
+        // required to avoid an error in the system
+        setControl(parent);
+        setPageComplete(false);
+
+        txtFolder.setText(ResourcesPlugin.getWorkspace().getRoot().getLocation().toOSString());
+    }
+
+    @Override
+    public boolean canFlipToNextPage() {
+        // single page
+        return false;
+    }
+
+    @Override
+    public boolean isPageComplete() {
+        return updateWorkspaceSelection();
+    }
+
+    /**
+     * Takes the selected/typed folder from the the {@link #txtFolder} and tries to obtain a valid Bnd Workspace. In any
+     * case, the {@link #tableViewer} is updated.
+     *
+     * @return true, when a Bnd Workspace was selected and properly initialized
+     */
+    private boolean updateWorkspaceSelection() {
+        final String selectedFolder = txtFolder.getText();
+        boolean result = false;
+        // check if folder containing a cnf-folder for Bnd was selected
+        if (null != selectedFolder && selectedFolder.trim().length() > 0) {
+            File chosenDirectory = new File(txtFolder.getText());
+            if (chosenDirectory.exists()) {
+                final Workspace bndWorkspace;
+                try {
+                    bndWorkspace = Workspace.getWorkspace(chosenDirectory);
+                    setErrorMessage(null);
+                    txtFolderErrorDecorator.hide();
+                    tableViewer.setInput(bndWorkspace.getAllProjects());
```

Comment:
Lets add the `cnf` folder to the tableViewer so we may need to add more than just bndWorkspace.getAllProjects() since that doesn't include the `cnf` project.